tasks/package.yaml

```diff
@@ -9,9 +9,9 @@
     - sudo_pkg
 
 - name: Install sudo for RedHat
-  yum:
-    name: sudo
-    state: present
+  action:
+    "{{ ansible_pkg_mgr }} pkg=sudo
+    state=present"
```

Comment:
I will do some improvements in my private fork (https://github.com/jtyr/ansible-sudo) so you can check that repo out once you switch to Ansible v2.0.